### PR TITLE
Bring back Verify page for direct links

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5008
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5002
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4995

Brings back Verify page as a gate for internal links asking user to verify their account.

Keeping the use of URLSearchParams as query params will be removed in V4 of react router
https://github.com/ReactTraining/react-router/issues/4410

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -15,23 +15,24 @@ require('babel-polyfill');
...
+  gi: './src/js/gi/gi-entry.jsx',
+  hca: './src/js/hca/hca-entry.jsx',
   letters: './src/js/letters/letters-entry.jsx',
+  messaging: './src/js/messaging/messaging-entry.jsx',
   pensions: './src/js/pensions/pensions-entry.jsx',
-  'post-911-gib-status': './src/js/post-911-gib-status/post-911-gib-status-entry.jsx',
-  'health-beta': './src/js/health-beta/health-beta-entry.jsx',
-  'pre-need': './src/js/pre-need/pre-need-entry.jsx',
+  rx: './src/js/rx/rx-entry.jsx',
+  verify: './src/js/login/verify-entry.jsx',
```
Comment (U-DON) [inline config/webpack.config.js]: What's with the ordering here?

File: src/js/login/components/Verify.jsx [hunk 2]
```diff
@@ -78,6 +94,7 @@ class Verify extends React.Component {
 }
 
 Verify.propTypes = {
+  shouldRedirect: PropTypes.bool,
```
Comment (U-DON) [inline src/js/login/components/Verify.jsx]: Where is `shouldRedirect` coming from?

File: config/webpack.config.js [hunk 3]
```diff
@@ -15,24 +15,25 @@ require('babel-polyfill');
...
+  hca: './src/js/hca/hca-entry.jsx',
   letters: './src/js/letters/letters-entry.jsx',
+  messaging: './src/js/messaging/messaging-entry.jsx',
   pensions: './src/js/pensions/pensions-entry.jsx',
-  'post-911-gib-status': './src/js/post-911-gib-status/post-911-gib-status-entry.jsx',
-  'health-beta': './src/js/health-beta/health-beta-entry.jsx',
-  'veteran-id-card': './src/js/veteran-id-card/veteran-id-card-entry.jsx',
-  'pre-need': './src/js/pre-need/pre-need-entry.jsx',
+  rx: './src/js/rx/rx-entry.jsx',
+  verify: './src/js/login/verify-entry.jsx',
```
Comment (U-DON) [inline config/webpack.config.js]: What's with the ordering?

File: src/sass/login.scss [hunk 4]
```diff
@@ -238,19 +238,24 @@ span.sidelines {
...
-      text-decoration: underline;
+    span {
+      &:hover {
+        color: $color-gold;
+        text-decoration: underline;
+      }
+
+      &.signin-spacer {
+        &:hover {
+          color: $color-white;
```
Comment (U-DON) [inline src/sass/login.scss]: Should this have `text-decoration: none` as well?

**Review by U-DON — APPROVED** [review]

File: config/webpack.config.js (hunk 3 above)
Comment (bshyong) [reply config/webpack.config.js]: I sorted these alphabetically

Comment (U-DON) [reply config/webpack.config.js]: But it goes `claims-status` to `veteran-id-card` and then starts over at `auth` again. Is there a secondary sort here?

**Pull request merged**